java/src/org/openqa/selenium/chrome/ChromeDriverService.java

```diff
@@ -283,34 +285,35 @@ protected void loadSystemProperties() {
     @Override
     protected List<String> createArgs() {
       List<String> args = new ArrayList<>();
-      args.add(String.format("--port=%d", getPort()));
+      try (Formatter formatter = new Formatter(Locale.US)) {
```

Comment:
It might be a good idea to add the same implementation to [GeckoDriver](https://github.com/SeleniumHQ/selenium/blob/trunk/java/src/org/openqa/selenium/firefox/GeckoDriverService.java) service as well to ensure consistency across languages.

Reply:
Will do so. 

However the fix still require some investigation for firefox, so I'm still looking into it and I'll commit the changes and Tests

Reply:
@pujagani  Added the required logic in GekoDriverService.java